shinano-common-caf/proprietary/Android.mk

```diff
@@ -41,7 +41,7 @@ include $(CLEAR_VARS)
 LOCAL_MODULE_TAGS   := eng
 LOCAL_MODULE        := tad_static
 LOCAL_MODULE_CLASS  := BIN
-LOCAL_SRC_FILES     := sbin/tad_static
+LOCAL_SRC_FILES     := vendor/sbin/tad_static
```

Comment:
Dunno why I still have it in /sbin but tad_static *need* to be into ramdisk and not system partition.